client/jetpack-connect/plans-landing.jsx

```diff
@@ -86,6 +88,12 @@ class PlansLanding extends Component {
 		this.props.recordTracksEvent( 'calypso_jpc_help_link_click' );
 	};
 
+	handleInfoButtonClick = info => {
+		this.props.recordTracksEvent( 'calypso_jpc_help_' + info + '_click', {
```

Comment:
Programmatically creating event names is not recommended - they should be hardcoded so they are easily searchable in the code base.

Reply:
yup!